src/domain/user/create-user.use-case.ts

```diff
@@ -0,0 +1,16 @@
+import { AppDataSource } from '../../data-source';
+import { User } from '../../data/db';
+import { UserModel } from '../../model';
+
+export async function createUserRow(data: UserModel) {
```

Comment:
nit: para a nomenclatura ficar correta
```suggestion
export async function createUserUseCase(data: UserModel) {
```